json-rpc/types/src/views.rs

```diff
@@ -609,9 +609,9 @@ impl MetadataView {
             self.script_hash_allow_list = Some(vm_publishing_option.script_allow_list);
             self.module_publishing_allowed = Some(vm_publishing_option.is_open_module);
         }
-        if let Some(diem_version) = diem_root.get_diem_version()? {
-            self.diem_version = Some(diem_version.major);
-        }
+        // if let Some(diem_version) = diem_root.get_diem_version()? {
```

Comment:
you have commented out the `get_diem_version` part

Reply:
Right. I moved the diem version fetching part from `with_diem_root` to get_metadata: https://github.com/diem/diem/blob/7429a2976b7b7e184992c405838bd78a394ebee0/json-rpc/src/data.rs#L64. I need to do the same move for verifying client.